CharlieBackend.Core/Entities/Theme.cs

```diff
@@ -12,5 +12,7 @@ public Theme()
         public string Name { get; set; }
 
         public virtual ICollection<Lesson> Lessons { get; set; }
+
+        public virtual ICollection<ScheduledEvent> ScheduledEvents { get; set; }
```

Comment:
Does theme actually need schedule?

Reply:
field added as a part of foreign keyconstraint